### PR TITLE
3.1 entity interface

### PR DESCRIPTION
**Pull request by skie**

### CODE REVIEW

File: src/View/Helper/FormHelper.php [hunk 1]
```diff
@@ -17,6 +17,7 @@
 use Cake\Collection\Collection;
 use Cake\Core\Configure;
 use Cake\Core\Exception\Exception;
+use Cake\Datasource\EntityInterface;
 use Cake\Form\Form;
 use Cake\ORM\Entity;
```
Comment (dereuromark) [inline src/View/Helper/FormHelper.php]: This use statement can then be removed?

**Comment by jadb** [comment]
@skie could you please squash commits so I can merge?

**Comment by lorenzo** [comment]
Git history lovers would probably hate me now

**Comment by ADmad** [comment]
promo-angry-mob

**Comment by lorenzo** [comment]
@ADmad haha

**Pull request merged**